src/metabase/cmd/copy.clj

```diff
@@ -164,6 +164,10 @@
       (map (fn [database]
              (cond-> database
                (= (:engine database) "h2") (assoc :details "{}")))))
+    :model/Setting
+    ;; Never create dumps with read-only-mode turned on.
+    ;; It will be confusing to restore from and prevent key rotation.
+    (remove (fn [{k :key}] (= k "read-only-mode")))
```

Comment:
@dpsutton 's solution to read-only dumps.

Reply:
I don't understand this comment. It seems like this remove xform on 170 will filter out the "read-only-mode" setting, but it will still be possible to create dumps with read-only-mode turned on, right? (it'll just not dump that setting).

I might be missing something, though

Reply:
Maybe I can parse the comment now, so this will not serialize the read-only-mode setting, and it will end up in the default (false) state. Would you consider just making it `false` in the xformer instead of removing it? That would be a little more direct.

Reply:
I'm not super confident that's strictly better, because it means committing some data to a written format and it also shouldn't be a factor for "normal" dumps. Since the end result is the same, I'd rather default on the conservative side and not add new data to dumps.